README.md

```diff
@@ -2,177 +2,232 @@
+option( EXIV2_ENABLE_DYNAMIC_RUNTIME  "Use dynamic runtime (used for static libs)"            OFF )
+option( EXIV2_ENABLE_WIN_UNICODE      "Use Unicode paths (wstring) on Windows"                OFF )
+option( EXIV2_ENABLE_CURL             "USE Libcurl for HttpIo"                                OFF )
+option( EXIV2_ENABLE_SSH              "USE Libssh for SshIo"                                  OFF )
+option( EXIV2_BUILD_SAMPLES           "Build sample applications"                             ON  )
+option( EXIV2_BUILD_PO                "Build translations files"                              OFF )
+option( EXIV2_BUILD_EXIV2_COMMAND     "Build exiv2 command-line executable"                   ON  )
+option( EXIV2_BUILD_UNIT_TESTS        "Build unit tests"                                      OFF )
+option( EXIV2_TEAM_EXTRA_WARNINGS     "Add more sanity checks using compiler flags"           OFF )
+option( EXIV2_TEAM_WARNINGS_AS_ERRORS "Treat warnings as errors"                              OFF )
+option( BUILD_WITH_CCACHE             "Use ccache to speed up compilations"                   OFF )
+```
+
+You specify and option with the syntax:
+
+```bash
+$ cmake .. -DBUILD_SHARED_LIBS=OFF .. -DEXIV2_ENABLE_XMP=OFF
+```
+
+<name id="1-4"></a>
+### 1.4) Build, test and install on _***ix**_ platforms (Linux, Cygwin and MacOS-X)
+On _***ix**_ p platforms such as Linux, Cygwin and MacOS-X, the recommended way to build Exiv2 is:
 
-Building and Installing
-=======================
 
-You can build the libraries in the following ways:
+```bash
+$ cd <exiv2dir>
+$ mkdir build
+$ cd build
+$ cmake ..
+$ cmake --build . --config Release
+$ make test
+$ sudo make install
+```
 
-1 Autotools: UNIX-like systems (including GNU/Linux, MacOS-X, Cygwin (32 and 64), MinGW (32 and 64)
-  - general notes follow
-  - FAQ concerning Cygwin/MSYS and Mac OS X:
-    http://dev.exiv2.org/projects/exiv2/wiki/FAQ
+##### Uninstall:
 
-2 Microsoft Visual C++ solutions
-  - see [msvc/ReadMe.txt](https://github.com/Exiv2/exiv2/blob/master/msvc/ReadMe.txt)      (32bit and 64bit builds Visual Studio 2005,08,10,12,13,15)
+```bash
+$ sudo make uninstall
+```
+
+<name id="1-5"></a>
+### 1.5) Build and install using Visual Studio
+
+See [README-CONAN](README-CONAN.md)
+
+<name id="1-6"></a>
+### 1.6) Building Exiv2 with CMake and conan
 
-3 CMake (support for all platforms/compilers except MinGW)
-  - see [README-CMAKE](https://github.com/Exiv2/exiv2/blob/master/README-CMAKE)
-    for more information
+See [README-CONAN](README-CONAN.md)
 
-To build a commercial version of the Exiv2 library, see also section
-"Commercial version" at the end of this file.
 
-On UNIX-like systems, use the GNU configure script. Run the following
-commands from the top directory (containing this file) to configure,
-build and install the library and utility:
 
-    $ ./configure
-    $ make
-    $ sudo make install  (Cygwin/MinGW $ make install)
+<name id="2"></a>
+## 2) Linking your own code with Exiv2
```

Comment:
I think this section has become obsoleted. We are still providing the pkg-config file, but we are also providing now a Config-file with CMake that should ease the consumption of Exiv2 in other projects configured with CMake.

I can take care of updating this section, once this PR gets merged. I have create the issue #359 to not forget about it.